docs/modules/ROOT/partials/getting-started/ref-registry-artifact-states.adoc

```diff
@@ -0,0 +1,39 @@
+// Metadata created by nebel
+// ParentAssemblies: assemblies/getting-started/as_registry-reference.adoc
+
+[id="registry-artifact-states"]
+
+= {registry} artifact states
+These are the valid artifact states in {registry}:
+
+.{registry} artifact states
+[%header,cols=2*]
+|===
+|State
+|Description
+|`ENABLED`
+|Basic state, all the operations are available.
+|`DISABLED`
+|The artifact and its metadata is viewable and searchable using the {registry} web console, but its content cannot be fetched by any client.
+|`DEPRECATED`
+|The artifact is fully usable but a header is added to the REST API response whenever the artifact content is fetched.
+|`DELETED`
+|Artifact not available through the REST API.
+|===
+
+.{registry} artifact state transitions
+[%header,cols=2*]
+|===
+|State
+|Valid Transition States
+|`ENABLED`
+|`DISABLED`,`DEPRECATED`,`DELETED`
+|`DISABLED`
+|`ENABLED`,`DEPRECATED`,`DELETED`
+|`DEPRECATED`
+|`ENABLED`,`DISABLED`,`DELETED`
+|`DELETED`
+|`NONE`
+|===
```

Comment:
Do we need this section at all?  We're basically documenting that any state can be transitioned to any other state.  Maybe not useful.  I'd much rather see an explanation of states and the motivation for having them at all.

Reply:
Right, I just realized that after writing this. This wasn't the situation, but now it is, I guess we can delete this section.